src/partitioned-dbs/index.rst

```diff
@@ -130,6 +136,33 @@ the query across all ``q`` shards in the database. This mean that you do
 not have to wait for all ``q`` shards to respond, which is both
 efficient and faster.
 
+In other words, you can use partitions to help pre-process views. As another
+example, you may be recording traffic to various websites, and 95% of your
+queries are based around 'what happened in relation to website example.com'.
+By partitioning your database with the key ``example.com:uuid``, and using
+the ``/dbname/_partition/example.com`` query, you are significantly simplifying
+the work the query engine has to do.
+
+When to NOT use partitions?
+===========================
+
+If you can not split your data using a common partition, or you end up with
+a small number of partitions, it is more efficient to use seperate databases.
+Following on with the example above, if you are only monitoring 10 websites,
+it is far more efficient to have 10 databases, rather than 1 database with
+10 partitions, with 10,000,000 entries in one partition, 500 in another, and
+10,000 in all the others.
```

Comment:
It might be more straightforward to talk about "consistently sized partitions" .